property/views.py

```diff
@@ -509,28 +509,23 @@ def get_queryset(self):
             (now() - datetime.timedelta(30)
              ).date(), '%Y-%m-%d'))
         date = dt.strptime(string_date, '%Y-%m-%d')
-        city = self.request.query_params['city']
+
+        city = self.request.query_params.get('city')
+
         query_results = Property.active_objects.filter(
-            address__City__icontains=city,
```

Comment:
@Tevinthuku any reason behind the use of dunders ( __ ) in the variable naming.

Reply:
Hey @alexxsanya , this is how django does the filtering checkout https://docs.djangoproject.com/en/2.2/topics/db/queries/#retrieving-specific-objects-with-filters . the `__` help access properties so for instance `city` is contained within `address` so that is why we use the `__`

Reply:
Okay @Tevinthuku.